### DISCUSSION TITLE
equalTo function only accepts a value.

### DISCUSSION THREAD

**Discussion by vamidi** [kreait/firebase-php#453] [Ideas]
**Is your feature request related to a problem? Please describe.**
I am trying to use the equalto function in kreait firebase, but it won't allow me to assign the optional key.
Normally in the firebase equalto function you can give optional key column. That is something I need as well.
I tried to look for solutions and stumbled on the `Query` class, but that one is only for internal use.

**Describe the solution you'd like**
It would be nice to see documentation about the equalTo function or maybe a solution for the key value assignment.

**Comment by jeromegamez** [comment]
Here's the documentation 😄

https://firebase-php.readthedocs.io/en/5.7.0/realtime-database.html#equalto

**Comment by vamidi** [comment]
Thanks @jeromegamez, but I can't add a key to the function what you normally can with the firebase sdk. 

`equalTo ( value :  number | string | boolean | null ,  key ? :  string ) : Query`

**Comment by jeromegamez** [comment]
Oh, I just found it in the JS SDK docs, this seems to have been added in-between, thanks for the heads up!

I will see how I can implement this in the SDK, but I'm mostly offline for the next 3 weeks, so I can't give you an ETA 😅

**Comment by vamidi** [comment]
Nicee! Great that you are going to work on it. Is there a way to get around this for now until you return?

[EDIT]
No worries anymore I found out that I just had a little bit of a design flaw in my database. Basically I wanted to check for the key to be a certain value, but I couldn't find anything, so I just added a member to the node with the key as value.

Thanks for updating the API and helping me out so quick!
